content/source/docs/enterprise/before-installing/postgres-requirements.html.md

```diff
@@ -7,37 +7,41 @@ page_title: "PostgreSQL Requirements - Before Installing - Terraform Enterprise"
 
 -> **Note:** These requirements apply to the external services operational mode, not the managed disk operational mode. See the [Pre-Install Checklist](./index.html) for more information.
 
-When Terraform Enterprise uses an external PostgreSQL database, the
-following must be present on it:
+To use an external PostgreSQL database with Terraform Enterprise, the following
+requirements must be met:
 
-* PostgreSQL version `>= 9.4` and `< 10.0`
-* User with the ability to create/modify/read tables and indices on all schemas created
-  * If it's not feasible to have a user with "CREATE EXTENSION", then create the [extensions](#extensions) below before installation
-* The following PostgreSQL schemas must be installed into the database: `rails`, `vault`, `registry`
+* The PostgreSQL Server version must be `>= 9.4` and `< 11.0`.
+* A PostgreSQL user must be created with the following permissions on the database:
+  * The ability to create, modify, and read all tables and indices on all schemas within the database. Usually this is granted if the user is an owner of the database.
+  * The ability to create extensions. If it is not feasible to have a user with the "CREATE EXTENSION" privilege, then refer to the [Creating Extensions](#creating-extensions) section below for information on creating the necessary extensions.
+* The `rails`, `vault`, and `registry` PostgreSQL schemas must be created on the database. These schemas will be automatically created by the Terraform Enterprise application if they do not already exist. To create the schemas manually, refer to the [Creating Schemas](#creating-schemas) section below.
 
 ## Creating Schemas
```

Comment:
I would change this section name to "Manually Creating Schemas" and then add a note about this is only required in versions below X

Reply:
Will do!